app/views/notification_mailer/digest.html.erb

```diff
@@ -1,25 +1,98 @@
+            <% if @type == :daily %>
+              Your Notifications From The Past 24h
+            <% elsif @type == :instant %>
+              Your Recent Notifications
+            <% end %>
+          </h2>
+        </td>
+      </tr>
+    </table>
+  </tr>
+</td>
 
-<% @notifications.each do |project, project_notifications| %>
-  <div class='project-wrapper'>
-    <div class='project-name-wrapper'>
-      <span class='project-name'><%= project.name %></span>
-      <hr>
-    </div>
+<tr>
+  <td>
+    <table cellspacing="0" cellpadding="0">
+      <% @notifications.each do |project, project_notifications| %>
+        <tr>
+          <td colspan="12">
+            <table cellspacing="0" cellpadding="0" align="center">
+              <tr>
+                <td height="20px"></td>
+              </tr>
+              <tr>
+                <% if defined?(Dradis::Pro) %>
+                  <td width="20px"></td>
+                  <td align="left" nowrap style="font-weight: 700; font-size: 1.1rem; line-height: 1.2rem;"><%= project.name %></td>
+                <% end %>
+                <td width="20px"></td>
+                <td style="width: 590px">
+                  <hr style="border-top: 0.5px solid #ddd; border-bottom: 0.5px solid #ddd; border-right: 0.5px solid transparent; border-left: 0.5px solid transparent;">
+                </td>
+                <td width="20px"></td>
+              </tr>
+            </table>
+          </td>
+        </tr>
 
-    <% project_notifications.each do |_, notifications| %>
-      <% presenter = DigestPresenter.new(notifications, project, self) %>
-      <%= render partial: 'notification_mailer/notification', locals: { presenter: presenter } %>
-    <% end %>
-  </div>
+        <tr>
+          <td colspan="12">
+            
+          </td>
+        </tr>
 
-  <hr>
+        
+        <% project_notifications.each do |_, notifications| %>
+        <tr>
+          <td>
+            <table cellspacing="0" cellpadding="20px">
+              <% presenter = DigestPresenter.new(notifications, project, self) %>
+              <%= render partial: 'notification_mailer/notification', locals: { presenter: presenter } %>
+            </table>
+          </td>
+        </tr>
+        <% end %>
 
-  <%= link_to 'See all notifications', project_notifications_url(project), class: 'button' %>
-<% end %>
+        <tr>
+          <td colspan="12">
+            <table cellspacing="0" cellpadding="0" align="center">
+              <tr>
+                <td width="20px"></td>
+                <td width="590px">
+                  <hr style="border-top: 0.5px solid #ddd; border-bottom: 0.5px solid #ddd; border-right: 0.5px solid transparent; border-left: 0.5px solid transparent;">
+                </td>
+                <td width="20px"></td>
+              </tr>
+            </table>
+          </td>
+        </tr>
+        
+        <% unless defined?(Dradis::Pro) %>
```

Comment:
Same here. No problem having a link to the notifications page. It does actually exist in CE and should work fine.

Reply:
✅